backend/src/main/java/gov/cdc/usds/simplereport/config/WebConfiguration.java

```diff
@@ -19,6 +19,7 @@
   public static final String HEALTH_CHECK = "/health";
   public static final String PATIENT_EXPERIENCE = "/pxp/**";
   public static final String ACCOUNT_REQUEST = "/account-request";
+  public static final String USER_ACCOUNT_REQUEST = "/user";
```

Comment:
Ended up going with `user-account`, since as the custom login work goes on we'll have more endpoints that are tied to accounts/authorization but not necessarily account creation.